src/fsharp/FSComp.txt

```diff
@@ -1,6 +1,6 @@
-# -----------------------------------------------------------------------------
-# use a completely new error number and add new messages at the end of the file
-# -----------------------------------------------------------------------------
+# -------------------------------------------------------------------------------
+# use a completely new error number and keep messages in their surrounding groups
```

Comment:
one reason for this is to reduce merge conflicts, when we append at the end, it always conflict, when kept in suroundings, it is unlikely to conflict.